pydicom/dataset.py

```diff
@@ -789,7 +793,13 @@ def _pixel_data_numpy(self):
                     #   in a Numpy error later on.
                     pass
 
-        pixel_array = numpy.fromstring(pixel_bytearray, dtype=numpy_dtype)
+        if self.BitsAllocated == 1:
+            # if single bits are used for binary representation, a uint8 array
+            # has to be converted to a binary-valued array (that is 8 times bigger)
+            pixel_array = numpy.unpackbits(numpy.fromstring(pixel_bytearray, dtype='uint8'))
+        else:
+            pixel_array = numpy.fromstring(pixel_bytearray, dtype=numpy_dtype)
```

Comment:
This will only work for the single segment datasets. Perhaps it is a good idea to add a check for the total number of segments (`SegmentSequence` size), and exit with warning that this type of data is not handled if it exceeds 1. Or add support for multiple segments, but need to be careful with separating individual frames, since need to account for bit order and that bit-encoded segments frames may not need to interrupt at byte boundary.